test/e2e/storage/testsuites/provisioning.go

```diff
@@ -837,14 +837,22 @@ func prepareSnapshotDataSourceForProvisioning(
 ) (*v1.TypedLocalObjectReference, func()) {
 	_, clearComputedStorageClass := SetupStorageClass(client, class)
 
-	ginkgo.By("[Initialize dataSource]creating a initClaim")
-	updatedClaim, err := client.CoreV1().PersistentVolumeClaims(initClaim.Namespace).Create(context.TODO(), initClaim, metav1.CreateOptions{})
-	framework.ExpectNoError(err)
+	if initClaim.ResourceVersion != "" {
+		ginkgo.By("Skipping creation of PVC, it already exists")
```

Comment:
oh nice, I didn't know about `ResourceVersion`, so this ensures that the object is not the spec to create the PVC but it's instead a PVC object that was returned by the API server